packages/calcite-components-react/README.md

```diff
@@ -85,6 +89,8 @@ function onUpdate(event) {
   setSliderValue(event.target.value);
 }
 
+// TODO: Get suggestions for what this should be replaced with
+
 // need to access the dom node to set custom event listeners for props that aren't strings / numbers
 // https://stenciljs.com/docs/react#properties-and-events
```

Comment:
The link is outdated, but managed to track down a [commit to the previous content](https://github.com/ionic-team/stencil-site/blob/9b3e02742c028f16404cdcab74b9861456446254/src/docs/framework-integration/react.md#properties-and-events). Based on that, we could replace the link with https://lit.dev/docs/frameworks/react/#why-are-wrappers-needed.